setup.cfg

```diff
@@ -27,5 +27,10 @@ keywords = toml, parser, serilization, deserialization, serdes
 setup_requires =
     pybind11~=2.5
 zip_safe = false
+packages = find:
+package_dir = =src
```

Comment:
`= =src`

Typo?

Reply:
Nope. To specify the root source directory for a given module or package, you need to provide the value in a map for the `package_dir` argument to `setup`. To specify a root source directory for all packages and modules, use `*`.

INI and TOML don't work well with * as the leading charcuterie in an unquoted string, so in setup.cfg it was decided that the empty string will replace it. Therefore, you can read it as the empty string before the second `=`, followed by "src/"

Reply:
Ah, I see. Thanks.